converter/src/main/java/gov/cms/qpp/conversion/Context.java

```diff
@@ -21,7 +21,7 @@
 	private Program program = Program.ALL;
 	private Set<QrdaScope> scope = EnumSet.noneOf(QrdaScope.class);
 	private boolean historical;
-	private boolean doDefaults = true;
+	private boolean doDefaults = false;
```

Comment:
Can we also make it so that the command line defaults `doDefaults` to false?  Currently it defaults it to true because of `CommandLineRunner.java:87` sets to use defaults when one _doesn't_ specify `-skipDefaults`.  I would prefer not have to specify `-skipDefaults` each time.

Reply:
sure